### ISSUE TITLE
Warning on project run with BHV: Property not found: `physics/3d/godot_physics/use_bvh`

### ISSUE THREAD

**Issue by Xrayez** [godotengine/godot#45142]
**Godot version:**
3.2.4 beta 8f265e8c0, likely 4.0.

**OS/device including version:**
Windows 10.

**Issue description:**
```
WARNING: ProjectSettings::_get: Property not found: physics/3d/godot_physics/use_bvh
     At: core\project_settings.cpp:209
```

**Steps to reproduce:**
It's likely because I compile the engine with `module_bullet_enabled=no`, which may effectively disable 3D physics, so no setting is initialized?

**Comment by akien-mga** [comment]
I guess `PhysicsServerSW` might be initialized before `World`.
```
servers/physics/physics_server_sw.cpp
1571:   bool use_bvh_or_octree = GLOBAL_GET("physics/3d/godot_physics/use_bvh");

scene/resources/world.cpp
340:    GLOBAL_DEF("physics/3d/godot_physics/use_bvh", true);
```

The warning will likely only happen once.

**Comment by lawnjelly** [comment]
Ah yes, sorry about that, I'll fix tomorrow. :+1:

(should not be a problem in the meantime aside from the warning I think).

**Comment by Xrayez** [comment]
Fixed by #45154.
